src/libstd/time/mod.rs

```diff
@@ -140,7 +140,7 @@ impl SystemTime {
     /// guaranteed to always be before later measurements (due to anomalies such
     /// as the system clock being adjusted either forwards or backwards).
     ///
-    /// If successful, `Ok(duration)` is returned where the duration represents
+    /// If successful, `Ok(Duration)` is returned where the duration represents
```

Comment:
Why `Duration` was capitalised? It is a variable, not a type. (type is `Result<Duration>`)